sdk/core/azure-core-amqp/src/amqp/management.cpp

```diff
@@ -249,5 +377,325 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
         throw std::runtime_error("Unknown management status."); // LCOV_EXCL_LINE
     }
   }
+#else
+
+  void ManagementClientImpl::OnMessageSenderStateChanged(
+      _internal::MessageSender const&,
+      _internal::MessageSenderState newState,
+      _internal::MessageSenderState oldState)
+  {
+    if (newState == oldState)
+    {
+      Log::Stream(Logger::Level::Error)
```

Comment:
Does the seriousness of these error levels correspond to the following?:
* **Error**: Logging level for failures that the application is unlikely to recover from.
* **Warning**: Logging level when a function fails to perform its intended task.
* **Informational**: Logging level when a function operates normally.
* **Verbose**: Logging level for detailed troubleshooting scenarios.

Reply:
Mostly they do. This one doesn't, but in general they do.